server/src/main/java/org/apache/druid/server/metrics/HistoricalMetricsMonitor.java

```diff
@@ -104,6 +106,8 @@ public boolean doMonitor(ServiceEmitter emitter)
       emitter.emit(builder.build("segment/count", count));
     }
 
+    emitter.emit(new ServiceMetricEvent.Builder().build("segment/totalCount", totalCount));
```

Comment:
I'm wondering whether the metrics visualization tool can compute this on its side. Is it hard to compute for some reason?